<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayStoDetailItemWindow</class>
 <widget class="QDialog" name="DisplayStoDetailItemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DisplayStoDetailItemWindow</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Container ID</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>711</width>
     <height>631</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Photo</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Container Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Use</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date Of Purchase</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total Number</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Storage Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Display DetailItem In Storage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Click</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Storage/Container</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DisplayStoDetailItemWindow</receiver>
   <slot>setContainerCombo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DisplayStoDetailItemWindow</receiver>
   <slot>display()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>DisplayStoDetailItemWindow</receiver>
   <slot>move()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>830</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setContainerCombo()</slot>
  <slot>display()</slot>
  <slot>move()</slot>
 </slots>
</ui>
